src/results-main.js

```diff
@@ -199,7 +199,7 @@
     var html = template(resp);
     setElectionInfo(resp.AllElections[0]);
     $lists.html(html);
-    if ($contestFilter.html().indexOf("select") === -1) {
+    if ($contestFilter.html().indexOf("select") === -1 && resp.AllElections[0].contests.length > 1) {
```

Comment:
So this is generally what I expected, one thing though.

If AllElections doesn't exist or doesn't have any items this would throw an error than it's possible this will throw an error. So we will want to make sure this is a little more robust.

See here for error.

![image](https://user-images.githubusercontent.com/1933400/70465332-b99d6680-1a8e-11ea-817a-deaa6dac436b.png)

This will get a little long, so it might be worth adding a variable, something like

```js
var hasOnly1Content = condition1 && condtion2...
```

Reply:
I updated per your suggestion in case AllElections does not exist. However setElectionPrecinctsReporting, setElectionUpdateDate, and pollCallback also access properties that would throw the same error if they did not exist. I did not attempt to handle those as I do not know the expected behavior in the case where # of Precincts or Update Date do not exist.

Reply:
sounds, good, we could open up a refactor issue if you want to take a stab at this.